docs/index.html

```diff
@@ -1 +1,35 @@
-Test
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Blog Post</title>
+  </head>
+  <body>
```

Comment:
I think in general you did a good job with formatting the post with the regular h1,h2,p… structure. For something like a Blog-Post you should/could add additional structure to the markup to make sure the Search engine is geting the most out of it. 

Have a look [here](https://www.hongkiat.com/blog/html-5-semantics/#aswift_2:~:text=Document%20Outline%20in%20HTML5)

I think of `<article>`, `<time>`, `<header>`, …

Reply:
Yep, added a better semantic structure.